api/src/main/kotlin/team/backend/common/logging/decorator/LoggingResponseDecorator.kt

```diff
@@ -0,0 +1,28 @@
+package team.backend.common.logging.decorator
+
+import org.reactivestreams.Publisher
+import org.slf4j.Logger
+import org.springframework.core.io.buffer.DataBuffer
+import org.springframework.http.server.reactive.ServerHttpResponse
+import org.springframework.http.server.reactive.ServerHttpResponseDecorator
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import team.backend.support.asString
+import java.io.ByteArrayOutputStream
+import java.nio.channels.Channels
+
+class LoggingResponseDecorator internal constructor(val log: Logger, delegate: ServerHttpResponse): ServerHttpResponseDecorator(delegate) {
+
+    override fun writeWith(body: Publisher<out DataBuffer>): Mono<Void> {
+        return super.writeWith(
+            Flux.from(body)
+                .doOnNext { buffer: DataBuffer ->
+                    if (log.isInfoEnabled) {
+                        val bodyStream = ByteArrayOutputStream()
+                        Channels.newChannel(bodyStream).write(buffer.asByteBuffer().asReadOnlyBuffer())
+                        log.info("{}: {} - {} : {}", "<<< Response", String(bodyStream.toByteArray()),
```

Comment:
Doesn't it have \n front and back?

Reply:
I didn't understand the intent of the question.

Reply:
@BAEKJungHo 
It's in the request, but don't you need \n in the response?

Reply:
@mangchhe Because when give a response, it's flat(json format is flatten).